### PR TITLE
Implement admin topic watch stream

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
 import("../pkg").then(async fluvioWasm => {
   var Fluvio = fluvioWasm.Fluvio;
   var Offset = fluvioWasm.Offset;
-  const fluvio = await Fluvio.connect("ws://localhost:3000")
-  const fluvio_2 = await Fluvio.connect("ws://localhost:3000")
+
+  const fluvio = await Fluvio.connect("ws://192.168.49.2:3030/api/v1/websocket/", "9003", "YOUR TOKEN HERE");
```
Comment (nacardin) [inline js/index.js:5]: Should not be hard-coded

**Review by nacardin — CHANGES_REQUESTED** [review]

File: js/index.js (hunk 1 above)
Comment (simlay) [reply js/index.js:5]: Yeah, these should be part of some integration suite.

File: src/admin.rs [hunk 2]
```diff
@@ -65,3 +88,41 @@ impl From<NativeFluvioAdmin> for FluvioAdmin {
         }
     }
 }
+
+macro_rules! impl_stream {
```
Comment (simlay) [inline src/admin.rs:92]: you fancy. We should figure out how to use this for the `PartitionConsumerStream`

**Comment by nicholastmosher** [comment]
I re-branched off of main and cherry picked just my changes.

File: src/admin.rs [hunk 3]
```diff
@@ -68,3 +68,99 @@ impl From<NativeFluvioAdmin> for FluvioAdmin {
         }
     }
 }
+
+#[cfg(feature = "unstable")]
```
Comment (morenol) [inline src/admin.rs:72]: I think that we need to add [cargoArgs](https://github.com/wasm-tool/rollup-plugin-rust#build-options) to the rollup config, to enable this feature

**Review by simlay — APPROVED** [review]
lgtm

File: src/admin.rs (hunk 3 above)
Comment (nicholastmosher) [reply src/admin.rs]: I have tried loading this into cloud and it was successfully using the code in "unstable" so I think we're okay?

**Pull request merged**